blockchain/block/block_deserializer.go

```diff
@@ -22,7 +36,7 @@ func (bd *Deserializer) FromBlockProto(pbBlock *iotextypes.Block) (*Block, error
 	if err := b.Header.LoadFromBlockHeaderProto(pbBlock.GetHeader()); err != nil {
 		return nil, errors.Wrap(err, "failed to deserialize block header")
 	}
-	if err := b.Body.LoadProto(pbBlock.GetBody()); err != nil {
+	if err := b.Body.LoadProto(pbBlock.GetBody(), bd.evmNetworkID); err != nil {
```

Comment:
Difference between `Deserializer.FromBlockProto` and `Block.ConvertFromBlockPb`?

Reply:
they are the same, looks like `Block.ConvertFromBlockPb` can be removed (only test code used it), will file a new PR to do this (and change `selp.LoadProto` to private func)

Reply:
The change is not large. Could you finish it in this pr?